DATA605/README.md

```diff
@@ -0,0 +1,224 @@
+
+
+<!-- toc -->
+
+- [DATA605 Class Projects](#data605-class-projects)
+  * [Choosing a project](#choosing-a-project)
+  * [Working](#working)
+  * [Pre-requisites](#pre-requisites)
+  * [Project assignment](#project-assignment)
+    + [Documentation](#documentation)
+    + [Submitting project](#submitting-project)
+  * [Example of a class project](#example-of-a-class-project)
+
+<!-- tocstop -->
+
+
+# DATA605 Class Projects
+
+- The goal of the class project is to learn cutting-edge modern big data
+  technology and write a (small) example of a system using it
+- Each class project is similar in spirit to the tutorials for various
+  technologies (e.g., Git, Docker, SQL, Mongo, Airflow, Dask) we have looked and
+  studied in classes
+
+## Choosing a project
+
+- Each student should pick one of the projects from the [signup sheet](https://docs.google.com/spreadsheets/d/1Ez5uRvOgvDMkFc9c6mI21kscTKnpiCSh4UkUh_ifLIw/edit?gid=0#gid=0)
+  - The difficulty of the project does affect the final grade, but we want to
+    give a way for everyone to select a project based on their level of computer
+    literacy
+  - You will need to give us some information including your GitHub name so we
```

Comment:
- An easy project should take around 2-3 full-days to complete and will yield a B or C as grade, if well-done
- A medium project should take around 4-5 full-days of work to complete and will yield a A or B as grade, if well-done
- A difficult project should take around 6-7 days of work to complete and will yield a A or A+ as grade, if well-done